content/troubleshooting/common-issues.md

```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
 **Solution**:
 * Confirm that you have set the correct bundle identifiers for all targets in Xcode.
 * Verify that the bundle identifier set in Xcode matches the bundle identifier set in the Flutter workflow editor under code signing.
+
+
+### Common Podfile issue
+The following issue is usually occurs when it comes to podfile issues:
+```
+The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99
+```
+
```

Comment:
Add here:
The error usually occurs if the deployment target set in Podfile is lower than required or when the Podfile is missing.